BENCH_DATA M3_M_COMP <Bench.TS_datasets.cTimeSeriesDatasetSpec object at 0x7ff1293034e0>
TIME :  Date N= 126 H= 18 HEAD= [0 1 2 3 4] TAIL= [121 122 123 124 125]
SIGNAL :  N1928 N= 126 H= 18 HEAD= [ 2908.  2978.  3474.  3730.  3644.] TAIL= [ 5594.  6338.  5800.  5402.  5902.]
Index   N1928  Date
1      2908.0     0
2      2978.0     1
3      3474.0     2
4      3730.0     3
5      3644.0     4
<class 'pandas.core.frame.DataFrame'>
Index: 126 entries, 1 to 126
Data columns (total 2 columns):
N1928    126 non-null float64
Date     126 non-null int64
dtypes: float64(1), int64(1)
memory usage: 3.0+ KB
None
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 140, in trainModel
    lAutoF1.train(df , self.mTSSpec.mTimeVar , iSignal, iHorizon)
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 25, in train
    self.mSignalDecomposition.train(iInputDS, iTime, iSignal, iHorizon, iExogenousData);
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 459, in train
    logger.info("START_TRAINING '" + iSignal + "'")
Message: "START_TRAINING 'N1928'"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 140, in trainModel
    lAutoF1.train(df , self.mTSSpec.mTimeVar , iSignal, iHorizon)
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 25, in train
    self.mSignalDecomposition.train(iInputDS, iTime, iSignal, iHorizon, iExogenousData);
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 481, in train
    logger.info("END_TRAINING_TIME_IN_SECONDS '" + iSignal + "' " + str(self.mTrainingTime))
Message: "END_TRAINING_TIME_IN_SECONDS 'N1928' 7.455905437469482"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 85, in getInfo
    logger.info("TIME_DETAIL " + self.mTrend.mTimeInfo.info());
Message: "TIME_DETAIL TimeVariable='Date' TimeMin=0 TimeMax=85 TimeDelta=1.0 Estimation = (0 , 86) Validation = (86 , 108) Test = (108 , 126) Horizon=18"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 86, in getInfo
    logger.info("SIGNAL_DETAIL " + self.info());
Message: "SIGNAL_DETAIL SignalVariable='_N1928' Min=2908.0 Max=6948.0  Mean=5356.42857143 StdDev=905.410425469"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 87, in getInfo
    logger.info("BEST_TRANSOFORMATION_TYPE '" + self.mTransformation.get_name("") + "'");
Message: "BEST_TRANSOFORMATION_TYPE '_'"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 88, in getInfo
    logger.info("BEST_DECOMPOSITION  '" + self.mOutName + "' [" + self.getFormula() + "]");
Message: "BEST_DECOMPOSITION  '_N1928_ConstantTrend_residue_zeroCycle_residue_AR(31)' [ConstantTrend + NoCycle + AR(31)]"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 89, in getInfo
    logger.info("TREND_DETAIL '" + self.mTrend.mOutName + "' [" + self.mTrend.mFormula + "]");
Message: "TREND_DETAIL '_N1928_ConstantTrend' [ConstantTrend]"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 90, in getInfo
    logger.info("CYCLE_DETAIL '"+ self.mCycle.mOutName + "' [" + self.mCycle.mFormula + "]");
Message: "CYCLE_DETAIL '_N1928_ConstantTrend_residue_zeroCycle' [NoCycle]"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 91, in getInfo
    logger.info("AUTOREG_DETAIL '" + self.mAR.mOutName + "' [" + self.mAR.mFormula + "]");
Message: "AUTOREG_DETAIL '_N1928_ConstantTrend_residue_zeroCycle_residue_AR(31)' [AR(31)]"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 92, in getInfo
    logger.info("MODEL_MAPE MAPE_Fit=" + str(self.mFitPerf.mMAPE) + " MAPE_Forecast=" + str(self.mForecastPerf.mMAPE)  + " MAPE_Test=" + str(self.mTestPerf.mMAPE) );
Message: 'MODEL_MAPE MAPE_Fit=0.0458 MAPE_Forecast=0.0641 MAPE_Test=0.047'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 93, in getInfo
    logger.info("MODEL_MASE MASE_Fit=" + str(self.mFitPerf.mMASE) + " MASE_Forecast=" + str(self.mForecastPerf.mMASE)  + " MASE_Test=" + str(self.mTestPerf.mMASE) );
Message: 'MODEL_MASE MASE_Fit=0.6368 MASE_Forecast=0.7293 MASE_Test=0.623'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 94, in getInfo
    logger.info("MODEL_L1 L1_Fit=" + str(self.mFitPerf.mL2) + " L1_Forecast=" + str(self.mForecastPerf.mL2)  + " L1_Test=" + str(self.mTestPerf.mL1) );
Message: 'MODEL_L1 L1_Fit=298.259219423 L1_Forecast=486.538446101 L1_Test=264.742944456'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 95, in getInfo
    logger.info("MODEL_L2 L2_Fit=" + str(self.mFitPerf.mL2) + " L2_Forecast=" + str(self.mForecastPerf.mL2)  + " L2_Test=" + str(self.mTestPerf.mL2) );
Message: 'MODEL_L2 L2_Fit=298.259219423 L2_Forecast=486.538446101 L2_Test=340.136365465'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 96, in getInfo
    logger.info("MODEL_COMPLEXITY " + str(self.getComplexity()) );
Message: 'MODEL_COMPLEXITY 31'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 97, in getInfo
    logger.info("AR_MODEL_DETAIL_START");
Message: 'AR_MODEL_DETAIL_START'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 98, in getInfo
    self.mAR.dumpCoefficients();
  File "/home/antoine/dev/python/packages/pyaf/TS/Scikit_Models.py", line 138, in dumpCoefficients
    logger.info("AR_MODEL_COEFF " + str(i) + " " + str(k) + " " + str(lDict[k]));
Message: 'AR_MODEL_COEFF 1 _N1928_ConstantTrend_residue_zeroCycle_residue_Lag1 0.542989022947'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 98, in getInfo
    self.mAR.dumpCoefficients();
  File "/home/antoine/dev/python/packages/pyaf/TS/Scikit_Models.py", line 138, in dumpCoefficients
    logger.info("AR_MODEL_COEFF " + str(i) + " " + str(k) + " " + str(lDict[k]));
Message: 'AR_MODEL_COEFF 2 _N1928_ConstantTrend_residue_zeroCycle_residue_Lag12 0.373648784963'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 98, in getInfo
    self.mAR.dumpCoefficients();
  File "/home/antoine/dev/python/packages/pyaf/TS/Scikit_Models.py", line 138, in dumpCoefficients
    logger.info("AR_MODEL_COEFF " + str(i) + " " + str(k) + " " + str(lDict[k]));
Message: 'AR_MODEL_COEFF 3 _N1928_ConstantTrend_residue_zeroCycle_residue_Lag24 0.280012020089'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 98, in getInfo
    self.mAR.dumpCoefficients();
  File "/home/antoine/dev/python/packages/pyaf/TS/Scikit_Models.py", line 138, in dumpCoefficients
    logger.info("AR_MODEL_COEFF " + str(i) + " " + str(k) + " " + str(lDict[k]));
Message: 'AR_MODEL_COEFF 4 _N1928_ConstantTrend_residue_zeroCycle_residue_Lag25 -0.222830578963'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 98, in getInfo
    self.mAR.dumpCoefficients();
  File "/home/antoine/dev/python/packages/pyaf/TS/Scikit_Models.py", line 138, in dumpCoefficients
    logger.info("AR_MODEL_COEFF " + str(i) + " " + str(k) + " " + str(lDict[k]));
Message: 'AR_MODEL_COEFF 5 _N1928_ConstantTrend_residue_zeroCycle_residue_Lag7 0.211977912904'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 98, in getInfo
    self.mAR.dumpCoefficients();
  File "/home/antoine/dev/python/packages/pyaf/TS/Scikit_Models.py", line 138, in dumpCoefficients
    logger.info("AR_MODEL_COEFF " + str(i) + " " + str(k) + " " + str(lDict[k]));
Message: 'AR_MODEL_COEFF 6 _N1928_ConstantTrend_residue_zeroCycle_residue_Lag13 -0.149289108238'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 98, in getInfo
    self.mAR.dumpCoefficients();
  File "/home/antoine/dev/python/packages/pyaf/TS/Scikit_Models.py", line 138, in dumpCoefficients
    logger.info("AR_MODEL_COEFF " + str(i) + " " + str(k) + " " + str(lDict[k]));
Message: 'AR_MODEL_COEFF 7 _N1928_ConstantTrend_residue_zeroCycle_residue_Lag31 0.144303093393'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 98, in getInfo
    self.mAR.dumpCoefficients();
  File "/home/antoine/dev/python/packages/pyaf/TS/Scikit_Models.py", line 138, in dumpCoefficients
    logger.info("AR_MODEL_COEFF " + str(i) + " " + str(k) + " " + str(lDict[k]));
Message: 'AR_MODEL_COEFF 8 _N1928_ConstantTrend_residue_zeroCycle_residue_Lag19 -0.138989982584'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 98, in getInfo
    self.mAR.dumpCoefficients();
  File "/home/antoine/dev/python/packages/pyaf/TS/Scikit_Models.py", line 138, in dumpCoefficients
    logger.info("AR_MODEL_COEFF " + str(i) + " " + str(k) + " " + str(lDict[k]));
Message: 'AR_MODEL_COEFF 9 _N1928_ConstantTrend_residue_zeroCycle_residue_Lag8 -0.107262087383'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 98, in getInfo
    self.mAR.dumpCoefficients();
  File "/home/antoine/dev/python/packages/pyaf/TS/Scikit_Models.py", line 138, in dumpCoefficients
    logger.info("AR_MODEL_COEFF " + str(i) + " " + str(k) + " " + str(lDict[k]));
Message: 'AR_MODEL_COEFF 10 _N1928_ConstantTrend_residue_zeroCycle_residue_Lag23 0.0878911895189'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M3Comp.py", line 27, in <module>
    tester5.run_multiprocessed()
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(processes=nbprocesses, maxtasksperchild=None)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 99, in getInfo
    logger.info("AR_MODEL_DETAIL_END");
Message: 'AR_MODEL_DETAIL_END'
Arguments: ()
   Transformation                                              Model  \
0          _N1928  _N1928_Lag1Trend_residue_zeroCycle_residue_AR(31)   
1          _N1928  _N1928_Lag1Trend_residue_bestCycle_byL2_residu...   
2          _N1928  _N1928_ConstantTrend_residue_bestCycle_byL2_re...   
3          _N1928  _N1928_ConstantTrend_residue_zeroCycle_residue...   
4          _N1928  _N1928_LinearTrend_residue_zeroCycle_residue_A...   
5          _N1928  _N1928_LinearTrend_residue_bestCycle_byL2_resi...   
6          _N1928  _N1928_PolyTrend_residue_bestCycle_byL2_residu...   
7          _N1928  _N1928_PolyTrend_residue_zeroCycle_residue_AR(31)   
8          _N1928    _N1928_Lag1Trend_residue_zeroCycle_residue_NoAR   
9    CumSum_N1928  CumSum_N1928_Lag1Trend_residue_zeroCycle_resid...   
10     Diff_N1928  Diff_N1928_Lag1Trend_residue_zeroCycle_residue...   
11   CumSum_N1928  CumSum_N1928_Lag1Trend_residue_bestCycle_byL2_...   
12   CumSum_N1928  CumSum_N1928_ConstantTrend_residue_zeroCycle_r...   
13   CumSum_N1928  CumSum_N1928_LinearTrend_residue_zeroCycle_res...   
14         _N1928  _N1928_Lag1Trend_residue_bestCycle_byL2_residu...   
15   CumSum_N1928  CumSum_N1928_PolyTrend_residue_zeroCycle_resid...   
16   CumSum_N1928  CumSum_N1928_Lag1Trend_residue_zeroCycle_resid...   
17   CumSum_N1928  CumSum_N1928_Lag1Trend_residue_bestCycle_byL2_...   
18     Diff_N1928  Diff_N1928_ConstantTrend_residue_zeroCycle_res...   
19     Diff_N1928  Diff_N1928_ConstantTrend_residue_bestCycle_byL...   
20   CumSum_N1928  CumSum_N1928_LinearTrend_residue_bestCycle_byL...   
21   CumSum_N1928  CumSum_N1928_PolyTrend_residue_bestCycle_byL2_...   
22   CumSum_N1928  CumSum_N1928_ConstantTrend_residue_bestCycle_b...   
23         _N1928  _N1928_LinearTrend_residue_zeroCycle_residue_NoAR   
24         _N1928  _N1928_LinearTrend_residue_bestCycle_byL2_resi...   
25     Diff_N1928  Diff_N1928_ConstantTrend_residue_zeroCycle_res...   
26     Diff_N1928  Diff_N1928_ConstantTrend_residue_bestCycle_byL...   
27     Diff_N1928  Diff_N1928_Lag1Trend_residue_zeroCycle_residue...   
28         _N1928    _N1928_PolyTrend_residue_zeroCycle_residue_NoAR   
29         _N1928  _N1928_PolyTrend_residue_bestCycle_byL2_residu...   
..            ...                                                ...   
34     Diff_N1928  Diff_N1928_LinearTrend_residue_zeroCycle_resid...   
35     Diff_N1928  Diff_N1928_LinearTrend_residue_bestCycle_byL2_...   
36     Diff_N1928  Diff_N1928_LinearTrend_residue_zeroCycle_resid...   
37     Diff_N1928  Diff_N1928_LinearTrend_residue_bestCycle_byL2_...   
38   CumSum_N1928  CumSum_N1928_LinearTrend_residue_zeroCycle_res...   
39   CumSum_N1928  CumSum_N1928_LinearTrend_residue_bestCycle_byL...   
40     Diff_N1928  Diff_N1928_PolyTrend_residue_zeroCycle_residue...   
41     Diff_N1928  Diff_N1928_PolyTrend_residue_bestCycle_byL2_re...   
42     Diff_N1928  Diff_N1928_PolyTrend_residue_zeroCycle_residue...   
43     Diff_N1928  Diff_N1928_PolyTrend_residue_bestCycle_byL2_re...   
44  RelDiff_N1928  RelDiff_N1928_ConstantTrend_residue_zeroCycle_...   
45  RelDiff_N1928  RelDiff_N1928_ConstantTrend_residue_bestCycle_...   
46  RelDiff_N1928  RelDiff_N1928_LinearTrend_residue_zeroCycle_re...   
47  RelDiff_N1928  RelDiff_N1928_PolyTrend_residue_zeroCycle_resi...   
48  RelDiff_N1928  RelDiff_N1928_LinearTrend_residue_bestCycle_by...   
49  RelDiff_N1928  RelDiff_N1928_PolyTrend_residue_bestCycle_byL2...   
50  RelDiff_N1928  RelDiff_N1928_ConstantTrend_residue_zeroCycle_...   
51  RelDiff_N1928  RelDiff_N1928_Lag1Trend_residue_zeroCycle_resi...   
52  RelDiff_N1928  RelDiff_N1928_ConstantTrend_residue_bestCycle_...   
53  RelDiff_N1928  RelDiff_N1928_Lag1Trend_residue_bestCycle_byL2...   
54  RelDiff_N1928  RelDiff_N1928_LinearTrend_residue_zeroCycle_re...   
55  RelDiff_N1928  RelDiff_N1928_PolyTrend_residue_zeroCycle_resi...   
56  RelDiff_N1928  RelDiff_N1928_LinearTrend_residue_bestCycle_by...   
57  RelDiff_N1928  RelDiff_N1928_PolyTrend_residue_bestCycle_byL2...   
58  RelDiff_N1928  RelDiff_N1928_Lag1Trend_residue_zeroCycle_resi...   
59  RelDiff_N1928  RelDiff_N1928_Lag1Trend_residue_bestCycle_byL2...   
60   CumSum_N1928  CumSum_N1928_PolyTrend_residue_zeroCycle_resid...   
61   CumSum_N1928  CumSum_N1928_PolyTrend_residue_bestCycle_byL2_...   
62   CumSum_N1928  CumSum_N1928_ConstantTrend_residue_zeroCycle_r...   
63   CumSum_N1928  CumSum_N1928_ConstantTrend_residue_bestCycle_b...   

    Complexity  FitCount        FitL1         FitL2  FitMAPE  FitMASE  \
0           63        86   245.156017    318.085165   0.0494   0.6990   
1           71        86   244.051332    320.010987   0.0493   0.6958   
2           39        86   236.861528    304.935973   0.0482   0.6753   
3           31        86   223.347267    298.259219   0.0458   0.6368   
4           47        86   227.118003    297.450843   0.0463   0.6476   
5           55        86   230.793660    299.322568   0.0469   0.6580   
6           55        86   219.943142    284.580622   0.0446   0.6271   
7           47        86   229.849452    300.167610   0.0467   0.6553   
8           32        86   346.651163    438.802811   0.0689   0.9884   
9           64        86   380.465116    543.702894   0.0803   1.0848   
10          64        86   346.651163    438.802811   0.0689   0.9884   
11          72        86   456.514440    763.912579   0.1028   1.3016   
12          63        86   414.842106    587.146205   0.0917   1.1828   
13          79        86   358.811030    477.340553   0.0736   1.0230   
14          40        86   344.499164    436.897344   0.0687   0.9822   
15          79        86   351.385415    441.390224   0.0702   1.0019   
16          95        86   433.054656    598.182349   0.0954   1.2347   
17         103        86   452.613067    606.266844   0.0987   1.2905   
18          63        86   711.167815    807.335885   0.1371   2.0277   
19          71        86   791.280280    883.733309   0.1529   2.2561   
20          87        86   503.278578    656.672947   0.1045   1.4349   
21          87        86   529.665964    796.558266   0.1106   1.5102   
22          71        86  1112.635584   2864.143493   0.2704   3.1723   
23          16        86   350.548699    441.119429   0.0722   0.9995   
24          24        86   352.369482    440.364821   0.0724   1.0047   
25          32        86  1038.334235   1134.887858   0.1970   2.9605   
26          40        86  1052.275650   1147.453349   0.2000   3.0002   
27          95        86   423.332659    515.356097   0.0806   1.2070   
28          16        86   360.043344    450.961490   0.0742   1.0266   
29          24        86   352.457568    439.083865   0.0729   1.0049   
..         ...       ...          ...           ...      ...      ...   
34          79        86   431.796575    530.760497   0.0898   1.2311   
35          87        86   437.738263    539.101668   0.0910   1.2481   
36          48        86   419.418558    534.535060   0.0823   1.1958   
37          56        86   423.568688    536.725761   0.0829   1.2077   
38          48        86   922.426034   1174.505918   0.1807   2.6300   
39          56        86  1176.436527   1560.693308   0.2392   3.3543   
40          79        86   511.776640    602.464712   0.1053   1.4592   
41          87        86   496.355667    587.602987   0.1023   1.4152   
42          48        86   404.579584    501.085749   0.0822   1.1535   
43          56        86   404.239936    501.260210   0.0821   1.1526   
44          32        86  2193.813953   2384.826199   0.4083   6.2550   
45          40        86  2193.813953   2384.826199   0.4083   6.2550   
46          48        86  2193.813953   2384.826199   0.4083   6.2550   
47          48        86  2193.813953   2384.826199   0.4083   6.2550   
48          56        86  2193.813953   2384.826199   0.4083   6.2550   
49          56        86  2193.813953   2384.826199   0.4083   6.2550   
50          63        86  2193.813953   2384.826199   0.4083   6.2550   
51          64        86  2193.813953   2384.826199   0.4083   6.2550   
52          71        86  2193.813953   2384.826199   0.4083   6.2550   
53          72        86  2193.813953   2384.826199   0.4083   6.2550   
54          79        86  2193.813953   2384.826199   0.4083   6.2550   
55          79        86  2193.813953   2384.826199   0.4083   6.2550   
56          87        86  2193.813953   2384.826199   0.4083   6.2550   
57          87        86  2193.813953   2384.826199   0.4083   6.2550   
58          95        86  2193.813953   2384.826199   0.4083   6.2550   
59         103        86  2193.813953   2384.826199   0.4083   6.2550   
60          48        86  1271.109692   3483.644005   0.3080   3.6242   
61          56        86  1478.958196   3491.567073   0.3444   4.2168   
62          32        86  7374.993240  22011.712392   1.7817  21.0276   
63          40        86  7859.840787  23303.125794   1.9200  22.4100   

    ForecastCount   ForecastL1    ForecastL2  ForecastMAPE  ForecastMASE  \
0              22   346.268930    436.993507        0.0597        0.7092   
1              22   362.476889    450.353893        0.0624        0.7423   
2              22   346.083882    492.935244        0.0628        0.7088   
3              22   356.121313    486.538446        0.0641        0.7293   
4              22   443.651950    608.279727        0.0803        0.9086   
5              22   443.855052    619.028700        0.0805        0.9090   
6              22   452.949759    624.681443        0.0814        0.9276   
7              22   465.790106    631.083896        0.0841        0.9539   
8              22   509.090909    597.306682        0.0861        1.0426   
9              22   509.090909    597.306682        0.0861        1.0426   
10             22   509.090909    597.306682        0.0861        1.0426   
11             22   514.558229    604.772166        0.0876        1.0538   
12             22   528.904542    619.001323        0.0886        1.0832   
13             22   536.166992    666.806153        0.0895        1.0981   
14             22   529.058229    613.183628        0.0896        1.0835   
15             22   563.670461    655.085485        0.0941        1.1544   
16             22   558.890956    630.434401        0.0942        1.1446   
17             22   556.643179    645.813744        0.0948        1.1400   
18             22   571.935400    771.084102        0.1004        1.1713   
19             22   587.374960    784.738658        0.1025        1.2029   
20             22   608.801188    780.085342        0.1032        1.2468   
21             22   645.966706    767.902242        0.1067        1.3229   
22             22   636.774351    788.191150        0.1082        1.3041   
23             22   677.881152    911.197624        0.1230        1.3883   
24             22   678.132804    914.855468        0.1232        1.3888   
25             22   785.454545    872.442205        0.1295        1.6086   
26             22   791.604872    879.698649        0.1305        1.6212   
27             22   858.766439    944.205219        0.1395        1.7587   
28             22   841.778155   1070.767366        0.1512        1.7239   
29             22   844.078516   1074.340867        0.1515        1.7287   
..            ...          ...           ...           ...           ...   
34             22  1119.062399   1237.905543        0.1807        2.2918   
35             22  1226.250028   1345.536011        0.1981        2.5113   
36             22  1432.386969   1509.217627        0.2316        2.9335   
37             22  1439.490183   1517.180379        0.2327        2.9480   
38             22  1532.768340   1613.799311        0.2461        3.1391   
39             22  1649.260847   1978.990413        0.2686        3.3777   
40             22  2067.586415   2246.095529        0.3444        4.2344   
41             22  2193.159158   2371.603263        0.3656        4.4915   
42             22  2297.694183   2426.864791        0.3815        4.7056   
43             22  2306.074096   2435.008396        0.3828        4.7228   
44             22  3166.727273   3215.554639        0.5166        6.4854   
45             22  3166.727273   3215.554639        0.5166        6.4854   
46             22  3166.727273   3215.554639        0.5166        6.4854   
47             22  3166.727273   3215.554639        0.5166        6.4854   
48             22  3166.727273   3215.554639        0.5166        6.4854   
49             22  3166.727273   3215.554639        0.5166        6.4854   
50             22  3166.727273   3215.554639        0.5166        6.4854   
51             22  3166.727273   3215.554639        0.5166        6.4854   
52             22  3166.727273   3215.554639        0.5166        6.4854   
53             22  3166.727273   3215.554639        0.5166        6.4854   
54             22  3166.727273   3215.554639        0.5166        6.4854   
55             22  3166.727273   3215.554639        0.5166        6.4854   
56             22  3166.727273   3215.554639        0.5166        6.4854   
57             22  3166.727273   3215.554639        0.5166        6.4854   
58             22  3166.727273   3215.554639        0.5166        6.4854   
59             22  3166.727273   3215.554639        0.5166        6.4854   
60             22  3620.626605   3788.380820        0.6185        7.4150   
61             22  3682.662437   4016.156063        0.6341        7.5420   
62             22  6074.727273   6100.323061        1.0000       12.4409   
63             22  6735.379371  10918.765085        1.1179       13.7939   

    TestCount       TestL1       TestL2  TestMAPE  TestMASE  
0          18   313.086890   390.376937    0.0554    0.7368  
1          18   324.287811   399.023416    0.0573    0.7631  
2          18   285.007526   362.096291    0.0508    0.6707  
3          18   264.742944   340.136365    0.0470    0.6230  
4          18   556.782056   619.546937    0.0991    1.3103  
5          18   555.148292   621.694150    0.0990    1.3064  
6          18   597.626027   674.178950    0.1059    1.4064  
7          18   602.231196   665.405739    0.1072    1.4172  
8          18   457.444444   529.854697    0.0805    1.0765  
9          18   457.444444   529.854697    0.0805    1.0765  
10         18   457.444444   529.854697    0.0805    1.0765  
11         18   541.156500   616.121406    0.0953    1.2735  
12         18   419.419134   538.657547    0.0730    0.9870  
13         18   512.645861   614.080410    0.0897    1.2064  
14         18   468.713871   534.445398    0.0828    1.1030  
15         18   469.923876   581.318857    0.0821    1.1059  
16         18   410.593127   533.542233    0.0718    0.9662  
17         18   501.790836   638.856044    0.0881    1.1808  
18         18  1087.975375  1140.138688    0.1934    2.5603  
19         18  1064.817378  1119.833731    0.1894    2.5058  
20         18   494.183450   613.938296    0.0871    1.1629  
21         18   636.353597   755.542518    0.1114    1.4975  
22         18   753.152004   875.224121    0.1330    1.7724  
23         18  1585.774043  1627.146381    0.2833    3.7317  
24         18  1584.834347  1627.790587    0.2832    3.7295  
25         18   386.369509   494.956465    0.0712    0.9092  
26         18   386.052288   496.526876    0.0711    0.9085  
27         18  1195.201746  1244.767541    0.2104    2.8126  
28         18  1924.631284  1961.476897    0.3430    4.5292  
29         18  1944.490891  1980.492608    0.3463    4.5759  
..        ...          ...          ...       ...       ...  
34         18  2192.929102  2304.269464    0.3841    5.1605  
35         18  2374.499546  2485.681330    0.4160    5.5878  
36         18  2420.128022  2491.848134    0.4221    5.6952  
37         18  2423.223447  2494.843323    0.4226    5.7025  
38         18  1134.445174  1187.835458    0.1960    2.6697  
39         18  1086.605103  1502.720847    0.1863    2.5571  
40         18  6255.030145  6513.917896    1.0987   14.7198  
41         18  6522.516391  6788.652500    1.1457   15.3492  
42         18  6155.060435  6355.931546    1.0791   14.4845  
43         18  6159.784213  6360.025915    1.0799   14.4956  
44         18  2787.000000  2809.155785    0.4873    6.5586  
45         18  2787.000000  2809.155785    0.4873    6.5586  
46         18  2787.000000  2809.155785    0.4873    6.5586  
47         18  2787.000000  2809.155785    0.4873    6.5586  
48         18  2787.000000  2809.155785    0.4873    6.5586  
49         18  2787.000000  2809.155785    0.4873    6.5586  
50         18  2787.000000  2809.155785    0.4873    6.5586  
51         18  2787.000000  2809.155785    0.4873    6.5586  
52         18  2787.000000  2809.155785    0.4873    6.5586  
53         18  2787.000000  2809.155785    0.4873    6.5586  
54         18  2787.000000  2809.155785    0.4873    6.5586  
55         18  2787.000000  2809.155785    0.4873    6.5586  
56         18  2787.000000  2809.155785    0.4873    6.5586  
57         18  2787.000000  2809.155785    0.4873    6.5586  
58         18  2787.000000  2809.155785    0.4873    6.5586  
59         18  2787.000000  2809.155785    0.4873    6.5586  
60         18  6436.324249  6476.297862    1.1377   15.1464  
61         18  6492.383809  6594.257072    1.1444   15.2783  
62         18  5695.000000  5705.875237    1.0000   13.4019  
63         18  4544.342593  8725.811362    0.8227   10.6941  

[64 rows x 18 columns]
  Transformation                                              Model  \
0         _N1928  _N1928_Lag1Trend_residue_zeroCycle_residue_AR(31)   
1         _N1928  _N1928_Lag1Trend_residue_bestCycle_byL2_residu...   
2         _N1928  _N1928_ConstantTrend_residue_bestCycle_byL2_re...   
3         _N1928  _N1928_ConstantTrend_residue_zeroCycle_residue...   
4         _N1928  _N1928_LinearTrend_residue_zeroCycle_residue_A...   

   Complexity  FitCount       FitL1       FitL2  FitMAPE  FitMASE  \
0          63        86  245.156017  318.085165   0.0494   0.6990   
1          71        86  244.051332  320.010987   0.0493   0.6958   
2          39        86  236.861528  304.935973   0.0482   0.6753   
3          31        86  223.347267  298.259219   0.0458   0.6368   
4          47        86  227.118003  297.450843   0.0463   0.6476   

   ForecastCount  ForecastL1  ForecastL2  ForecastMAPE  ForecastMASE  \
0             22  346.268930  436.993507        0.0597        0.7092   
1             22  362.476889  450.353893        0.0624        0.7423   
2             22  346.083882  492.935244        0.0628        0.7088   
3             22  356.121313  486.538446        0.0641        0.7293   
4             22  443.651950  608.279727        0.0803        0.9086   

   TestCount      TestL1      TestL2  TestMAPE  TestMASE  
0         18  313.086890  390.376937    0.0554    0.7368  
1         18  324.287811  399.023416    0.0573    0.7631  
2         18  285.007526  362.096291    0.0508    0.6707  
3         18  264.742944  340.136365    0.0470    0.6230  
4         18  556.782056  619.546937    0.0991    1.3103  
18 0     5772.0
1     5574.0
2     5578.0
3     6100.0
4     5420.0
5     5040.0
6     5332.0
7     5478.0
8     5694.0
9     5646.0
10    5148.0
11    5452.0
12    5614.0
13    5754.0
14    5484.0
15    5944.0
16    5482.0
17    5250.0
Name: N1928, dtype: float64
18 0     6075.290576
1     6460.747845
2     6185.900263
3     6750.844163
4     5981.656655
5     5562.189087
6     6369.905773
7     6182.827439
8     6705.382136
9     6488.635252
10    6343.896037
11    6348.945245
12    6167.306163
13    6530.833966
14    6356.165536
15    6734.473440
16    6270.352192
17    6088.270035
Name: N1928_Forecast, dtype: float64
FORECAST_DETAIL_ACTUAL M3_M_COMP N1928 [ 5772.  5574.  5578.  6100.  5420.  5040.  5332.  5478.  5694.  5646.
  5148.  5452.  5614.  5754.  5484.  5944.  5482.  5250.]
FORECAST_DETAIL_PREDICTED M3_M_COMP N1928 [ 6075.29057577  6460.74784522  6185.90026256  6750.84416283  5981.65665455
  5562.18908707  6369.90577292  6182.82743939  6705.3821355   6488.63525176
  6343.89603653  6348.94524466  6167.30616331  6530.83396588  6356.16553566
  6734.47344039  6270.35219151  6088.27003514]
BENCHMARK_PERF_DETAIL M3_M_COMP N1928 126 18 8.29392957687378 ConstantTrend + NoCycle + AR(31) 18 0.1395 0.1297 2.6072 768.978988926 796.539709084 -8.65485918855
BENCHMARK_SUCCESS 'M3_M_COMP_N1928_18'
